web/src/types/Assertion.types.ts

```diff
@@ -1,72 +1,43 @@
 import {CompareOperator} from 'constants/Operator.constants';
-import {LOCATION_NAME} from 'constants/Span.constants';
```

Comment:
I wanted to use interfaces but I found out those don't support property lookup so I decided to use types instead.